geode-core/src/main/java/org/apache/geode/management/internal/cli/remote/CommandExecutor.java

```diff
@@ -152,6 +161,17 @@ protected Object invokeCommand(Object command, GfshParseResult parseResult) {
         return cc;
       });
     }
+
+    Map<String, CacheConfig> configMap = ccService.getGroups()
```

Comment:
I don't quite see how this code is reachable. If we have `groups` passed in, then the above block is run. If no groups are set then the group is defaulted to `cluster` and the above block is run. Maybe I'm missing something?

Reply:
After all the groups specified (or "cluster") are cycled through in the `for` loop we drop into this code. The `*Command` implementor can override either `updateGroupConfig` or `updateAllConfigs`. If `updateGroupConfigs` is not overridden, then nothing gets processed in the `for` loop and everything gets processed via this `Map`. See the `AlterAsyncEventQueueCommand` changes for an `updateAllConfigs` implementation.

Reply:
Doh - I missed that the return was inside the closure and not inside the for loop. NM.